.github/workflows/pr-content-extraction.yml

```
@@ -18,15 +25,10 @@ jobs:
-      - name: Checkout repository
-        uses: actions/checkout@v6
+      - uses: ./.github/actions/setup
         with:
-          fetch-depth: 1  # Shallow clone for speed
-
-      - name: Setup Node.js
-        uses: actions/setup-node@v6
-        with:
-          node-version: '22'
+          fetch-depth: 1 # Shallow clone - only need current commit
+          install-deps: false # Don't install deps - we only use github-script
```

actionlint (1.7.9)

input "install-deps" is not defined in action "Setup" defined at "./.github/actions/setup". available inputs are "fetch-depth", "install-args", "node-version", "pnpm-version"

(action)